README.md

```diff
@@ -612,14 +647,19 @@ module.exports = {
 
 ### `implementation`
 
-Type: `Function | String`
+Type:
+
+```ts
+type implementation = require('postcss') | string
```

Comment:
It can be function, let's improve this, `require('postcss')` is default value

Reply:
`require('postcss')` exports a type file called `postcss.d.ts`. That is why I gave `require('postcss')`

Reply:
🤔  There is no type for package? Shall I leave it generic as `object`?

Reply:
I think we can do it, just add comment about it should be have the same API as postcss